src/postamble_minimal.js

```diff
@@ -189,6 +189,7 @@ WebAssembly.instantiate(Module['wasm'], imports).then(function(output) {
 #else
   /*** ASM_MODULE_EXPORTS ***/
 #endif
+  wasmTable = asm['__indirect_function_table'];
```

Comment:
this may be fine for now, but optimally we wouldn't have such a manual assignment? it should work like any other export. In particular, I'm not sure if export name minification works on it when written manually?

Reply:
Are you saying that we should drop the current/explict `wasmTable` globals and just switch to using `__indirect_function_table` everywhere instead?  And really on the export mechanism to ensure this exists?

I think that could work, but maybe as a followup?  Right now the normal export mechanism only really works for functions I think.  It assume that the exports are callable, but we can/should change that anyway,.